charts/sda-svc/values.yaml

```diff
@@ -225,7 +225,7 @@ global:
           jku: "https://login.elixir-czech.org/oidc/jwk"
 
   elixir:
-    oidcdHost: "https://login.elixir-czech.org/oidc/"
+    provider: "https://login.elixir-czech.org/oidc/"
```

Comment:
not it redirects to https://profile.aai.lifescience-ri.eu/, but should we make the switch :thinking: